CHANGES.txt

```diff
@@ -5,9 +5,14 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fix: Certain subselects with nested ``order by`` and ``limit`` or ``offset``
```

Comment:
maybe re-phrase this more towards users?
If they're rewritten or not isn't really relevant - they produced the wrong result.